sdh-proxy.c

```diff
@@ -65,9 +65,18 @@ int do_exit = 0;
 // Toggled with -d on command line. Displays debug info. 
 int debug = 0;
 
+// Toggled with -l on command line. Write Stats to stat.log
+int logstat = 0;
+char logstatfile[] = "sdh.stat";
+
 // Does nothing at the moment. Will enable rewriting subnet info if relevant. 
 int do_network_rewrite = 0; // Rewrite IP broadcast address for the new
                             // network interface
+int32_t pkt_rx = 0;
```

Comment:
How many packets did you see at your event? Did an int32 get close to overflowing? Probably better to make it an int64 just in case?